src/helpers/htmlTemplate.ts

```diff
@@ -0,0 +1,30 @@
+import type { Options } from 'qr-code-styling'
+
+const qrCodeStyling =
```

Comment:
We have a React here, so maybe we can put all this stuff into something, that `parses` the html?

Reply:
Not sure about it, had similar troubles before

Reply:
Unfortunately, I didn't come up with a way to do this directly using React, because we are translating our React code into static html, and the qr is created dynamically using javascript